src/main/kotlin/org/wycliffeassociates/otter/jvm/device/audio/AudioPlayer.kt

```diff
@@ -10,6 +10,7 @@ import org.wycliffeassociates.otter.common.device.IAudioPlayer
 import org.wycliffeassociates.otter.common.device.IAudioPlayerListener
```

Comment:
For all these files that aren't part of your new stuff, but may have been changed during a rebase or something, try checking out the copies on dev: `git checkout dev -- /src/main/kotlin/org/wycliffeassociates/otter/jvm/device/audio/AudioPlayer.kt` (or the whole folder, if you don't need any changes in it). Once you've checked out all the files you don't want to change, commit and push and they shouldn't show up in this pr anymore